src/test/java/bio/terra/service/dataset/DatasetDaoTest.java

```diff
@@ -84,7 +89,8 @@ private UUID createDataset(DatasetRequestModel datasetRequest, String newName) t
 
     private UUID createDataset(String datasetFile) throws Exception  {
         DatasetRequestModel datasetRequest = jsonLoader.loadObject(datasetFile, DatasetRequestModel.class);
-        return createDataset(datasetRequest, datasetRequest.getName() + UUID.randomUUID().toString());
+        return createDataset(datasetRequest, datasetRequest.getName() + UUID.randomUUID().toString(),
+            GoogleRegion.DEFAULT_GOOGLE_REGION.toString());
```

Comment:
The way this enum uses `toString` is really more purposeful than converting to a string. It really means, get the google name for this region so we can call a google API. Maybe `getName()` would be a better function name than `toString`.

Reply:
That's a good point - that was confusing to me when I first looked at his code. I've switched it over, although I hope I caught all of the cases since toString() should still be a valid method & wouldn't necessarily throw an error.